### PR TITLE
Add rustdoc team meeting

### PR DESCRIPTION
**Pull request by camelid**
cc @GuillaumeGomez

### CODE REVIEW

File: dev-tools.toml [hunk 1]
```diff
@@ -3,5 +3,6 @@ description = "Meetings for the dev tools team and its sub-teams"
 
 [meta]
 includes = [
-  "clippy.toml"
+  "clippy.toml",
+  "rustdoc.toml",
```
Comment (davidtwco) [inline dev-tools.toml:7]: Can you add it to `README.md` too?

File: rustdoc.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+uid = "1704823643569"
+title = "Rustdoc meeting"
+description = "Monthly team meeting of the Rustdoc team"
+location = "#t-rustdoc/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/t-rustdoc.2Fmeetings)"
+last_modified_on = "2024-01-09T18:15:00.00Z"
+start = "2024-02-12T20:00:00.00Z"
+end = "2024-02-12T21:00:00.00Z"
+status = "confirmed"
+organizer = { name = "rustdoc", email = "rustdoc@rust-lang.org" }
+recurrence_rules = [ { frequency = "monthly", by_day = [ { day = "monday", num = 2 } ] } ]
```
Comment (davidtwco) [inline rustdoc.toml:14]: Did you test this or would you like me to before merging?

Comment (camelid) [reply rustdoc.toml:14]: I did, and it works correctly 👍

File: rustdoc.toml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name = "Rustdoc Team"
+description = "Meetings for the Rustdoc team"
+
+[[events]]
+uid = "1704823643569"
+title = "Rustdoc meeting"
+description = "Monthly team meeting of the Rustdoc team"
+location = "#t-rustdoc/meetings on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/t-rustdoc.2Fmeetings)"
+last_modified_on = "2024-01-09T18:15:00.00Z"
+start = "2024-02-12T20:00:00.00Z"
```
[the remark this answers is not in the document]
Comment (GuillaumeGomez) [reply rustdoc.toml:10]: I think UTC is the simplest for setting the time. As for daylight saving, as the team prefers. When setting the date on zulip I set it depending on my local time so it followed european time but that was just a default, so not great.

**Comment by davidtwco** [comment]
CI failure likely just reflects need for a rebase to include other changes that fix that issue

File: rustdoc.toml (hunk 3 above)
[the remark this answers is not in the document]
Comment (camelid) [reply rustdoc.toml:10]: For me personally, after the DST switch in March I won't be able to attend the meeting anymore (if we do UTC) due to a time conflict.

**Review by davidtwco — APPROVED** [review]

**Pull request merged**